board/ubirch1r03/board.c

```diff
@@ -45,3 +46,62 @@ void board_install_bootloader_hook(void) {
 void NMI_Handler(void) {
   runBootloader(NULL);
 }
+
+volatile uint32_t lptmrCounter = 0U;
+volatile bool button_press_status = false;
+
+void BOARD_LPTMR_HANDLER(void)
+{
+  LPTMR_ClearStatusFlags(LPTMR0, kLPTMR_TimerCompareFlag);
+  lptmrCounter++;
```

Comment:
check if you really need this counter, if you set the lptimer to run for exactly one second, you can just check if it stopped.